index.html

```diff
@@ -0,0 +1,132 @@
+</head>
+<body>
+  <div class="page-wrapper">
+    <section class="work">
+      <div class="work-container">
+        <div class="card-work">
+          <div class="snapshoot"><img src="./images/Snapshoot_Portfolio.png" alt=""></div>
+          <div class="primary-text">
+            <h1>Tonic</h1>
+            <img src="./images/Frame2.png" alt="">
+            <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
+          </div>
+          <div>
+            <ul class="tags">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <div class="normal-button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <div class="work-container">
+        <div class="card-work">
+          <div class="snapshoot"><img src="./images/Snapshoot_Portfolio(1).png" alt=""></div>
+          <div class="primary-text">
+            <h1>Multi-Post <br>Stories</h1>
+            <img src="./images/Frame2.png" alt="">
+            <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
+          </div>
+          <div>
+            <ul class="tags">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <div class="normal-button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <div class="work-container">
+        <div class="card-work">
+          <div class="snapshoot"><img src="./images/Snapshoot_Portfolio(2).png" alt=""></div>
+          <div class="primary-text">
+            <h1>Tonic</h1>
+            <img src="./images/Frame2.png" alt="">
+            <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
+          </div>
+          <div>
+            <ul class="tags">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <div class="normal-button">
+            <button type="button">See Project</button>
+          </div>
+        </div>
+      </div>
+
+      <div class="work-container">
+        <div class="card-work">
+          <div class="snapshoot"><img src="./images/Snapshoot_Portfolio(3).png" alt=""></div>
+          <div class="primary-text">
+            <h1>Multi-Post <br>Stories</h1>
+            <img src="./images/Frame2.png" alt="">
+            <p>A daily selection of privately <br>personalized reads; no accounts or <br>sign-ups required.</p>
+          </div>
+          <div>
+            <ul class="tags">
+              <li>html</li>
+              <li>css</li>
+              <li>javaScript</li>
+            </ul>
+          </div>
+          <div class="normal-button">
+            <button type="button">See Project</button>
```

Comment:
- [ ] You did an amazing job on this project but please add the button interactions following [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Here you have a screenshot of button interactions:

![image](https://user-images.githubusercontent.com/74506933/159130385-b550b258-2bdb-4465-a17b-68dc363a34f2.png)

Reply:
Hello @Donard97, this is not part of the project requirements as mentioned in the project outline "Works and About Myself" sections

Reply:
Hello, @YussufOB thank you for tagging me. In fact, every button should have its interactions because it was on the design and it is a part of the project requirements to stick to the design as much as you can. Keep coding 💪